### DISCUSSION TITLE
ISV program discount activated to wrong Organization

### DISCUSSION THREAD

**Discussion by KoushikCU** [community/community#49607] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello, 

I've activated the discount coupon from Microsoft ISV success program to the incorrect organization URL which i have created. however, there was an existing Github account and Organization url for which the discount coupon to be activated.

Now that the discount is already activated in the duplicate url, can it be transferred to the actual? .

**Comment by alexgenne** [comment]
Hi,
did you solved your issue?

This is exactly what happend to me.
I already created an account and within that account i created the Enterprise and Organization. Then i used this account to redeem the discount. The discount is displayed within the personal account, but not within the enterprise account.

Would be great, if you could share the solution if there is one.

Regards
Alexander

**Reply by KoushikCU** [reply, marked as answer]
Hi Alex,

Yes, the issue got solved. I had raised this concern with the GitHub Support team and with their assistance from the back-end, and the Sponsership sub was transferred to the right account.

Regards,
Koushik

**Reply by alexgenne** [reply]
Thanks, that's what I did too

**Reply by Braytiner** [reply]
Hi! That happened to me too.

I tried to activate the discount coupon again, but I get an error message: Coupon can't be redeemed any more times.

I've opened a ticket for the billing team and I'm waiting for a solution.
